warhol.plugin.zsh

```diff
@@ -8,140 +8,102 @@ PLUGIN_BIN="$(dirname $0)/bin"
 export PATH=${PATH}:${PLUGIN_BIN}
 
 if [ "$TERM" != dumb ] && builtin type -p grc >/dev/null 2>&1; then
+  setopt complete_aliases
   alias colourify="$GRC -es --colour=auto"
 
-  # Use functions so we can still take advantage of ZSH completion functions
+  alias sudo="sudo "
```

Comment:
Why add this alias for `sudo`? It isn't color-related.

Reply:
You right, isn't directly for color, but Alias only works for the first command in the shell, by sample : sudo ping, only alias for sudo is resolved.

from man page :  http://www.linuxcommand.org/lc3_man_pages/aliash.html

>A trailing space in VALUE causes the next word to be checked for alias substitution when the alias is expanded.

__Src__ : 
I found a good post about it : https://stackoverflow.com/questions/37209913/how-does-alias-sudo-sudo-work :